.github/ISSUE_TEMPLATE/system-team-qg-checklist.md

```diff
@@ -0,0 +1,136 @@
+---
+name: System Team Quality Gate Checklist
```

Comment:
Since the TRGs are project wide, I would omit the 'system team' part. In my opinion it's just a Quality Gate Checklist

Reply:
[chore: removed system team as name](https://github.com/eclipse-tractusx/.github/pull/1/commits/ed6531e8c9d65f8c418d15d917f6498aaec9ce1b)